gui/wxpython/location_wizard/wizard.py

```diff
@@ -171,14 +162,26 @@ def __init__(self, wizard, parent, grassdatabase):
         self.locTitle = ''
 
         # text controls
-        self.tgisdbase = self.MakeStaticText(grassdatabase, size=(-1, -1))
-        self.tlocation = self.MakeTextCtrl("newLocation", size=(300, -1))
+        self.tgisdbase = self.MakeTextCtrl(grassdatabase, size=(400, -1), style = wx.TE_READONLY | wx.TRANSPARENT_WINDOW)       
+        self.tlocation = self.MakeTextCtrl("newLocation", size=(400, -1))
         self.tlocation.SetFocus()
         self.tlocation.SetValidator(
             GenericValidator(
                 grass.legal_name,
                 self._nameValidationFailed))
         self.tlocTitle = self.MakeTextCtrl(size=(400, -1))
+        
+        # text for required options
+        self.required_txt = self.MakeLabel("*")
```

Comment:
As I mentioned, don't use self here if it's used only locally within the scope of the method, if you make it an attribute of the class it looks like it's being used elsewhere too, which is confusing.